jax/_src/interpreters/pxla.py

```diff
@@ -457,22 +458,27 @@ def process_map(self, map_primitive, fun, tracers, params):
     return map(partial(MapTracer, self), out, outaxes)
 
   def process_custom_jvp_call(self, prim, fun, jvp, tracers, *, symbolic_zeros):
-    bind = HashableFunction(
-        lambda *args, **kwargs: prim.bind(
-            fun, jvp, *args, symbolic_zeros=symbolic_zeros, **kwargs),
-        (prim, fun, jvp, symbolic_zeros))
-    fake_primitive = FakePrimitive(multiple_results=True, bind=bind)
-    return self.process_primitive(fake_primitive, tracers, {})
+    if symbolic_zeros:
+      msg = "Please open an issue at https://github.com/google/jax/issues !"
+      raise NotImplementedError(msg)
```

Comment:
Say what it is that isn't implemented, in case it isn't obvious from the stack trace?

Reply:
Done!